components/script/Cargo.toml

```diff
@@ -65,7 +65,7 @@ indexmap = "1.0.2"
 ipc-channel = "0.11"
 itertools = "0.8"
 jstraceable_derive = {path = "../jstraceable_derive"}
-js = {package = "mozjs", version = "0.10.1"}
+js = {package = "mozjs", git = "https://github.com/servo/rust-mozjs", rev = "357d1db91e4e49c3f148fe05fdfae5f8f0834ca8"}
```

Comment:
Can we publish mozjs and go back to getting it from crates.io?

Reply:
Nope! https://github.com/rust-lang/cargo/issues/6917